pages.fr/common/b2sum.md

```diff
@@ -0,0 +1,28 @@
+# b2sum
+
+> Calcule les sommes de contrôle cryptographiques BLAKE2.
+> Plus d'informations : <https://www.gnu.org/software/coreutils/b2sum>.
+
+- Calcule la somme de contrôle BLAKE2 d'un fichier :
+
+`b2sum {{chemin/vers/fichier}}`
+
+- Calcule les sommes de contrôle BLAKE2 pour plusieurs fichiers :
+
+`b2sum {{chemin/vers/fichier1}} {{chemin/vers/fichier2}}`
+
+- Calcule la somme de contrôle BLAKE2 depuis stdin :
+
+`{{commande}} | b2sum`
+
+- Lit un fichier contenant des sommes de contrôle BLAKE2 et des noms de fichier et vérifie que tous les fichiers ont des sommes de contrôle correspondantes :
```

Comment:
```suggestion
- Lis un fichier contenant des sommes de contrôle BLAKE2 et des noms de fichier et vérifie que tous les fichiers ont des sommes de contrôle correspondantes :
```

We rather like imperative form than present form ;) 
https://github.com/tldr-pages/tldr/blob/1bb198bc9483ddbb607e515843cc85eba846342a/contributing-guides/style-guide.md#imperative-mood